### ISSUE TITLE
Ability to move selected faces from cluster to other cluster

### ISSUE THREAD

**Issue by illnesse** [pulsejet/memories#78]
So instead of deleting from current cluster, recrawl & redetect as new cluster etc... could we simply switch selected faces to a different (correct) cluster? Not sure how feasible that is or if it's better done on recognize side

**Comment by pulsejet** [comment]
From an API perspective, this is easy since we can move an image from one cluster to another with recognize's webdav API. The UI is more challenging. Ideally this should somehow go into either `viewer` (or some kind of plugin for it), or the sidebar where the user can select the particular face and re-categorize it.

**Comment by illnesse** [comment]
Why not reuse the UI from merging clusters? Workflow should be kind of identical from user perspective

Select faces from within cluster, go to menu, select new cluster, done

**Comment by pulsejet** [comment]
Yes, I misunderstood what you're referring to. That said, the feature I mentioned above (albeit separate) might also be useful.

**Comment by pulsejet** [comment]
![image](https://user-images.githubusercontent.com/10709794/196575095-f83cb19c-2c81-4d94-9202-377c4032bafa.png)
